### PR TITLE
Simplify and refactor SNZ pulse

### PR DESCRIPTION
**Pull request by andrea-pasquale**
This PR implements a simplification of the SNZ pulse, which will be used by qibocal to perform the leakage optimization.
I still need to test this PR on hardware. 

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibolab/pulses.py [hunk 1]
```diff
@@ -468,41 +468,43 @@ class SNZ(PulseShape):
     """
     Sudden variant Net Zero.
     https://arxiv.org/abs/2008.07411
+    (Supplementary materials)
```
Comment (Jacfomg) [inline src/qibolab/pulses.py:471]: Should we reference the specific image as well ?

**Review by Jacfomg — APPROVED** [review]
I feel new pulses should be tested on a oscilloscope on the lab before being used but that is to test the drivers.

File: src/qibolab/pulses.py (hunk 1 above)
Comment (andrea-pasquale) [reply src/qibolab/pulses.py:471]: Is it possible to reference an image in sphinx?

Comment (Jacfomg) [inline src/qibolab/pulses.py:471]: maybe this would be enough

```suggestion
    (Supplementary materials: FIG. S1.)
```

Comment (Jacfomg) [reply src/qibolab/pulses.py:471]: I don't know

**Pull request merged**